clientlibs/js/src/UpgradeClient.ts

```diff
@@ -225,16 +226,7 @@ export default class UpgradeClient {
     return response;
   }
 
-/**
- * Will return all the experiment conditions for the user.
- * Internally this uses the `context` and `userId` to query conditions for all eligible decision points at enrolling experiments for this user.
- *  
- * @example
- * ```typescript
- * const allExperimentConditionsResponse: IExperimentAssignmentv4[] = await upgradeClient.getAllExperimentConditions(workingGroup);
- * ```
- */
-  async getAllExperimentConditions(): Promise<IExperimentAssignmentv4[]> {
+  async getAllExperimentConditions(): Promise<IExperimentAssignmentv5[]> {
```

Comment:
Can someone who made the TS Client Lib updates for v5 also update the TSDoc comments/examples so that they accurately reflect to library user (and the testers 😉 ) how the v5 functions are designed to work?

Reply:
Added TSDoc for `getAllExperimentConditions()`